.container/Dockerfile

```diff
@@ -14,3 +14,7 @@ WORKDIR /app/camel
 RUN poetry install --with dev,docs -E all
 
 RUN pre-commit install
+
+RUN rm -rf /app/camel
```

Comment:
Hi @WHALEEYE, although volumes are provided, retaining the original framework may maintain the integrity of the image. What do you think?

Reply:
Sure, but I guess we should place the original framework into some other paths like `/temp`?

Reply:
I think we may choose not to move it, because if the user chooses to mount the volume, there will be two source codes in the container; but if the user does not choose to mount the volume, it is also feasible to run it directly without causing any problems. Users experience location confusion.

Reply:
Sure, this makes sense